### ISSUE TITLE
[Classic] [Winterspring] [Minor]: Zone changes to Alterac Valley, while in Winterspring

### ISSUE THREAD

**Issue by r00ty-tc** [dalaranwow/dalaran-wow#1973]
Hi,

There's a cave, with a furbolg chief inside at location 70,38 in Winterspring (close to the furbolg village). When you enter the cave, the zone (for the purposes of General/Local Defense) changes to Alterac Valley.

Possibly it's some zone/map flag on the server files? I'm not sure. Either way, cosmetic and not a serious issue. But, I thought I'd report it.

Video available if needed. But, I think just going to those co-ordinates and entering the cave is enough.

**Comment by Rushor** [comment]
Client-side issue. This cannot be fixed serverside

**Comment by r00ty-tc** [comment]
It's a moot point, as I'll get into. BUT! I disagree. If it was client side you'd see the zone/area change text appear on screen, as happens even when you make minor changes, let alone suddenly change a whole zone away. ONLY the channel changes, and this is done server side.

As you know since I reported this I've got a little bit into the scene and I actually went down to breaking down where it was going wrong. I never actually found the cause but this definitely changes server side. Perhaps one day it'll bother me enough to really find out what's going on. I'd actually forgotten about this old gem.

The main question that needs to be asked is whether it's a data problem (and as such the server correctly reports it) and whether this happened on retail. I never saw this reported ever on retail so I'm assuming not.

Not a serious problem and I agree it should stay closed. One day I might look at it from the TC side of things though.

There are other places this happens.. The plaguelands zones near Stratholme has some odd spots where you'll suddenly be in another zone channel. They're MUCH harder to find though.
